OperatingSystem-Services/Service-SOC/BasicDocumentation/DockerInstall-Wazuh.md

```diff
@@ -0,0 +1,48 @@
+# How to install Docker For Wazuh Purposes
+
+## DISCLAIMER: Do not follow docker docs, follow provided steps
+
+
+Follow these steps:
+
+## Increase max_map_count on your Docker host:
+
+`sudo sysctl -w vm.max_map_count=262144`
+
+### Or to do this permenantly do: 
+
+Edit `/etc/sysctl.conf` file, 
+
+Add `vm.max_map_count=262144` to the end of the file
+
+Run `sysctl vm.max_map_count` to make sure the number is 262144
+
+## Check Kernel Version
+
+Run: `sudo uname -r`
+
+Make sure kernel version is greater than 3.10
```

Comment:
What part of the output is the version number. For example I run it and get the output `6.5.6-300.fc39.x86_64` Point out the version is `6.5.6` (Or something).